### DISCUSSION TITLE
736

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#916] [Comments]
# 736

올해 2월부터 준비한 인프콘이 2023.08.15 에 성황리에 마무리 되었다. 인프콘의 모든 발표는 인프런에 무료로 공유되어 있다. 회고 써야지 써야지 하다가, 이제야 쓴다. 인프콘 컨퍼런스 운영에 대해서는 관여를 하지 않았다. TF 멤버분들이 100% 진행해주셔서 감사할 따름이다. 그래서 인프콘을 어떻게 준비했는지 그 과정이 궁금하신 분들은 아래 회고들을 참고해보면 좋다. 인프콘 2023 현장 스케치 인프콘 TF 회고 인프콘 TF 리드 - 앨리스의 회고 이번 후기는 인프콘 사이트 개발과 개인 발표자로서의 회고이다. 인프콘 사이트 준비 작년 인프콘 준비 기간 동안 개발 리소스가 거의 없어서, 공식 사이트 개발을 DevOps와 디자이너 두명이서 워드프레스로 테마 골라서 만들자로 결론내고 진행했었다. 그리고..

https://jojoldu.tistory.com/736

**Comment by mystyle2006** [comment]
수고하셨습니다! :) 올해는 참석에 실패했지만 내년에도 기회가 있기를!

**Comment by Sn-Kinos** [comment]
>그리고 공식 사이트의 여러 스펙을 확정하는 회의때 작업자들의 아이디어 대부분을 반려했다.
>
>여러 욕심 나는 아이디어들이 많았지만, 우리 같이 작은 스타트업에서 이정도 규모의 인력이 몇달간 행사 페이지에만 올인할 수는 없었다. 
>하고 싶어도 못한다. 
>버그나 장애등으로 인프콘 준비에 완전히 몰입할 수 없기 때문이다.
>
>그래서 인프콘 TF 멤버들을 다 모은 자리에서 기본 기능만 먼저 구현하자고 했다. 

비즈니스 속도를 0순위로 놓고 이를 기반으로 사고 및 의사결정이 진행된 다음, 구현을 1순위로 해서 작업을 이어나간 덕분에 프로세스를 무사히 마치고 여유시간을 확보하여 +@ 를 이어나가는 프로세스가 인상깊습니다! :D

**Comment by jimbaemon** [comment]
성장기회의 평등함을 주셔서 감사합니다 :)
